File: 1133-Rest-of-a-Division.py
# Read the input values
X = int(input())
Y = int(input())

# Ensure X is less than Y for ascending order
if X > Y:
    X, Y = Y, X

# Iterate over the range and check the condition
for number in range(X + 1, Y):
    if number % 5 == 2 or number % 5 == 3:
        print(number)

File: 1133-Rest-of-a-Division.cpp
#include <iostream>
using namespace std;

int main()
{
    int X, Y;

    // Read the input values
    cin >> X >> Y;

    // Ensure X is less than Y for ascending order
    if (X > Y)
    {
        int temp = X;
        X = Y;
        Y = temp;
    }

    // Iterate over the range and check the condition
    for (int number = X + 1; number < Y; number++)
    {
        if (number % 5 == 2 || number % 5 == 3)
        {
            cout << number << endl;
        }
    }

    return 0;
}
